iot/google/cloud/iot_v1/gapic/device_manager_client.py

```diff
@@ -235,11 +235,13 @@ def create_device_registry(self,
             metadata = []
         metadata = list(metadata)
         try:
-            routing_header = google.api_core.gapic_v1.routing_header.to_grpc_metadata(
-                [('parent', parent)], )
-            metadata.append(routing_header)
+            routing_header = [('parent', parent)]
         except AttributeError:
             pass
```

Comment:
This exception *cannot* happen:  no attribute access occurs, because `parent` itself is the desired string.

Reply:
lol